third_party/nvfuser/csrc/transform_rfactor.cpp

```diff
@@ -210,6 +210,33 @@ class ReplayRFactor : public ReplayTransformations {
     }
   }
 
+  void handle(Resize* resize) override {
```

Comment:
Is this used? I think we never rfactor a pad/slice consumer?

Reply:
You're right. I guess it was needed in some early prototype. Changed to error out if this hits. 1b5c23ccee